keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -149,22 +149,17 @@ metadata:
   finalizers:
     - cosi.io/finalizer [2]
 spec:
-  protocol:
-    name: [3]
-    version: [4]
-  bucketPrefix: [5]
-  bucketClassName: [6]
-  bucketInstanceName: [7]
+  bucketPrefix: [3]
+  bucketClassName: [4]
```

Comment:
Fwiw, I find "bucketClass" a fairly confusing term, particularly as at least S3 and GCS have "storage class" as a well-defined term.

Your definition:

>  The BucketClass (BC) defines a retention policy, driver specific parameters, and the provisioner name.

also implies ... a lot that is expressible in the APIs, but not the kind of thing I'd expect as a "string".

What's an example of what this is supposed to look like for "bucket lock on, dual-region GCS bucket over the JSON API"?

Reply:
The `BucketClass` is similar to k8s `StorageClass` but without may of the SC fields that are related to file and blocks, eg. _mountOptions_, _allowVolumeExpansion_, and some reserved key in _parameters_. Like a k8s SC, a COSI BC has no spec field and is immutable.